from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Na(object):
    def setupUi(self, Na):
        Na.setObjectName("Na")
        Na.setFixedSize(791, 461)
        Na.setMouseTracking(False)
        Na.setStyleSheet("background: rgb(255, 255, 255)")
        self.centralwidget = QtWidgets.QWidget(Na)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(200, 20, 131, 111))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(330, 40, 411, 81))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setItalic(False)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(50, 20, 141, 111))
        font = QtGui.QFont()
        font.setFamily("Arial")
        font.setPointSize(72)
        font.setBold(True)
        font.setWeight(75)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(50, 130, 241, 281))
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(310, 120, 411, 271))
        self.label_5.setObjectName("label_5")
        Na.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(Na)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 799, 25))
        self.menubar.setObjectName("menubar")
        Na.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(Na)
        self.statusbar.setObjectName("statusbar")
        Na.setStatusBar(self.statusbar)

        self.retranslateUi(Na)
        QtCore.QMetaObject.connectSlotsByName(Na)

    def retranslateUi(self, Na):
        _translate = QtCore.QCoreApplication.translate
        Na.setWindowTitle(_translate("Na", "나트륨"))
        self.label.setText(_translate("Na", "Na"))
        self.label_2.setText(_translate("Na", "나트륨"))
        self.label_3.setText(_translate("Na", "11."))
        self.label_4.setText(_translate("Na", "<html><head/><body><p><span style=\" font-size:14pt; font-weight:600;\">원자 번호  : 11번</span></p><p><span style=\" font-size:14pt; font-weight:600;\">원자 기호  : Na</span></p><p><span style=\" font-size:14pt; font-weight:600;\">영어 이름  : Sodium</span></p><p><span style=\" font-size:14pt; font-weight:600;\">한글 이름  : 나트륨</span></p><p><span style=\" font-size:14pt; font-weight:600;\">전자  수   : 11개</span></p><p><span style=\" font-size:14pt; font-weight:600;\">양성자 수  : 11개</span></p></body></html>"))
        self.label_5.setText(_translate("Na", "<html><head/><body><p><span style=\" font-size:14pt; font-weight:600;\">원자량     : 22.9897(g/mol)</span></p><p><span style=\" font-size:14pt; font-weight:600;\">밀도       : 0.971(g, Cm^3)</span></p><p><span style=\" font-size:14pt; font-weight:600;\">녹는점     : 97.8˚C = 208.04˚F = 370.95K</span></p><p><span style=\" font-size:14pt; font-weight:600;\">끓는점     : 882.95˚C = 1621.31˚F = 1156.1K</span></p><p><span style=\" font-size:14pt; font-weight:600;\">물질의 상태: 고체</span></p></body></html>"))



if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Na = QtWidgets.QMainWindow()
    ui = Ui_Na()
    ui.setupUi(Na)
    Na.show()
    sys.exit(app.exec_())
